reanalysis/metamist_registration.py

```diff
@@ -14,62 +14,14 @@
 from cpg_utils import to_path
 from cpg_utils.config import get_config
 
-from sample_metadata.apis import AnalysisApi
-from sample_metadata.model.analysis_type import AnalysisType
-from sample_metadata.model.analysis_model import AnalysisModel
-from sample_metadata.model.analysis_status import AnalysisStatus
-from sample_metadata.model.analysis_query_model import AnalysisQueryModel
-from sample_metadata.model.analysis_update_model import AnalysisUpdateModel
-
-
-def inactivate_old_entry(name: str, metadata: dict[str, str], anal_type: AnalysisType):
```

Comment:
this inactivation isn't necessary; the active flag in metamist doesn't currently behave correctly so it's a bit pointless